### PR TITLE
Tutorial explorer: preview mode on hourofcode.com

### PR DESCRIPTION
**Pull request by breville**
This brings the Tutorial Explorer preview mode, first introduced in https://github.com/code-dot-org/code-dot-org/pull/43313, to https://staging.hourofcode.com/learn?preview=true, since the original URL for the mode was deprecated in https://github.com/code-dot-org/code-dot-org/pull/53916.

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/public/learn/index.haml [hunk 1]
```diff
@@ -21,22 +21,27 @@ social:
...
-  - tutorial[:name]                                 = hoc_s(prefix + "name")
-  - tutorial[:shortdescription]                     = hoc_s(prefix + "shortdescription")
-  - tutorial[:longdescription]                      = hoc_s(prefix + "longdescription")
-  - tutorial[:language]                             = hoc_language(tutorial[:languages_supported])
-  - tutorial[:string_detail_grades]                 = hoc_s(prefix + "string_detail_grades")
-  - tutorial[:string_platforms]                     = hoc_s(prefix + "string_platforms")
-  - tutorial[:string_detail_platforms]              = hoc_s(prefix + "string_detail_platforms")
-  - tutorial[:string_detail_programming_languages]  = hoc_s(prefix + "string_detail_programming_languages")
+  -# Unless showing a preview, look up localized versions of strings.
+  - unless request.params["preview"]
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/hourofcode.com/public/learn/index.haml]: nit: can you pull this out into a variable, ie `preview = request.params["preview"]`. At least for me, that would be more readable

File: pegasus/sites.v3/hourofcode.com/public/learn/index.haml [hunk 2]
```diff
@@ -21,22 +21,27 @@ social:
...
+    - tutorial[:name]                                 = hoc_s(prefix + "name")
```
Comment (bethanyaconnor) [inline pegasus/sites.v3/hourofcode.com/public/learn/index.haml:38]: are these already set and being overwritten in this block? I guess I'm asking why there isn't an `else`

**Review by bethanyaconnor — APPROVED** [review]
a nit and a question but lgtm!

File: pegasus/sites.v3/hourofcode.com/public/learn/index.haml (hunk 1 above)
Comment (breville) [reply pegasus/sites.v3/hourofcode.com/public/learn/index.haml]: Good call, will do.

File: pegasus/sites.v3/hourofcode.com/public/learn/index.haml (hunk 2 above)
Comment (breville) [reply pegasus/sites.v3/hourofcode.com/public/learn/index.haml:38]: Correct.  The `cdo-tutorials` gsheet should still have English versions of these strings, which we overwrite with localized versions unless using the preview.

**Pull request merged**